src/components/OSCALSystemImplementationInventoryItems.js

```diff
@@ -80,7 +81,10 @@ export default function OSCALOSCALSystemImplementationTableTitleInventoryItems(
   return (
     <>
       <OSCALSystemImplementationTableTitle variant="h6" id="tableTitle">
-        Inventory Items
+        <OSCALAnchorLinkHeader
+          title="Inventory Items"
+          style={{ "padding-top": "1em", "padding-bottom": "1em" }}
+        />
```

Comment:
If we're consistently applying this style, can we just do that for `OSCALAnchorLinkHeader`?

Reply:
This padding style will be something consistent for only the `system-implementation` header styling, which will occur three times. I was trying to decide if this would be something I would want to make into a separate style that would be included in `OSCALTableStyles`.

Reply:
I definitely want it de-duplicated somehow